faucet.js

```diff
@@ -66,20 +72,33 @@ class Faucet {
      * @param dapp The address of dapp 
      */
     async checkAppliable(dapp) {
-        let r, sponsorInfo, faucetParams;
+        if(dapp === null) return {flag: false, message:''};
```

Comment:
Better to add error message, e.g. "contract address not specified"

Reply:
error message added